src/pages/loading-waiting-states/progress-bar/index.tsx

```diff
@@ -255,44 +248,41 @@ export const ProgressBar = (): JSX.Element => {
-                                            <>
+                                    <div>
+                                        <InfoListItem
+                                            classes={{
+                                                subtitle: classes.subTitle,
+                                            }}
+                                            key={`infolist${item.id}`}
+                                            backgroundColor={Colors.black[900]}
+                                            fontColor={Colors.black[50]}
+                                            color={Colors.black[50]}
+                                            title={item.name}
+                                            subtitle={item.status}
+                                            icon={<Description />}
+                                            iconAlign="left"
+                                            iconColor={Colors.black[200]}
+                                            rightComponent={
                                                 <Button
                                                     variant="outlined"
-                                                    className={classes.closeButtonContainer}
+                                                    className={classes.buttonContainer}
                                                     onClick={(): void => removeListItem(item.id, item.status)}
                                                 >
                                                     {item.progress === 100 ? 'View' : 'Cancel'}
                                                 </Button>
-                                            </>
-                                        }
-                                        message={
-                                            <div className={classes.messageContainer}>
-                                                <div className={classes.iconContainer}>
-                                                    <Description className={classes.icon} />
-                                                </div>
-                                                <div className={classes.messageTextContainer}>
-                                                    <Typography variant={'subtitle1'} color={'inherit'}>
-                                                        {item.name}
-                                                    </Typography>
-                                                    <Typography variant={'subtitle2'} className={classes.subTitle}>
-                                                        {item.status}
-                                                    </Typography>
-                                                </div>
-                                            </div>
-                                        }
-                                    />
+                                            }
+                                        />
```

Comment:
Since we switched to using an InfoListItem here, I'm going to suggest we go with the theme wrapper approach that we had discussed before. It greatly simplifies the styling — you just have to set the backgroundColor on the snackbarRoot and that's it...no prop changes, no classes, no extra styles:

```tsx
<Snackbar
    classes={{ root: classes.snackbarRoot }}
    open={item.open}
    autoHideDuration={item.progress === 100 ? 3000 : null}
    onClose={(e, reason): void => handleRequestClose(e, reason, item.id)}
    TransitionProps={{ timeout: 300, onExited: (): void => handleExited(item.id) }}
>
    <div>
        <MuiThemeProvider theme={createTheme(PXBThemes.blueDark)}>
            <InfoListItem
                title={item.name}
                subtitle={item.status}
                icon={<Description />}
                rightComponent={
                    <Button
                        variant="outlined"
                        style={{ width: 80 }}
                        onClick={(): void => removeListItem(item.id, item.status)}
                    >
                        {item.progress === 100 ? 'View' : 'Cancel'}
                    </Button>
                }
            />
            <LinearProgressWithLabel value={item.progress} />
        </MuiThemeProvider>
    </div>
</Snackbar>
```

The top one is the current implementation, bottom is the suggested with the theme wrapper:
![image](https://user-images.githubusercontent.com/29152776/126388162-feb3d12c-ed08-4fa2-b380-5bc11562b4cf.png)